src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj

```diff
@@ -107,7 +107,7 @@
     <PackageReference Include="Serilog.Sinks.File" Version="4.0.0" />
     <PackageReference Include="MallardMessageHandlers" Version="0.2.0-dev.21" />
     <PackageReference Include="Refit" Version="4.7.51" />
-    <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="8.6.2" />
+    <PackageReference Include="FluentValidation.DependencyInjectionExtensions" Version="9.5.4" />
```

Comment:
Do you need to update FluentValidation in Chinook.DynamicMvvm.FluentValidation?

Reply:
version 9.5.4 is what Chinook.DynamicMvvm.FluentValidation is using. I just updated the project to the same version